docs/harfbuzz-sections.txt

```diff
@@ -607,6 +607,12 @@ hb_ot_math_get_min_connector_overlap
 hb_ot_math_get_glyph_assembly
 </SECTION>
 
+<SECTION>
+<FILE>hb-ot-metadata</FILE>
+hb_ot_metadata_reference_entry
+hb_ot_metadata_t
```

Comment:
I personally try putting types before their uses.  Not sure if that's done consistently.